packages/cli/src/getHost.ts

```diff
@@ -0,0 +1,30 @@
+import internalIp from 'internal-ip'
+
+import { getTunnelAddress } from './tunnel'
+
+export enum HostType {
```

Comment:
nit: i personally don't like `enums` 😄 What makes the difference between  enum implementation and this -
```ts
const HostTypes = {
  lan : 'lan',
  localhost : 'localhost',
  tunnel : 'tunnel',
} as const
```
Because i (on the surface) don't find any, would gladly hear any `pros` of enums :smile: